frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -8,6 +8,24 @@ import { isDefaultClass } from '../storage-class';
 
 /* Component StorageClassDropdown - creates a dropdown list of storage classes */
 
+const getUnorderedItems = (items, filter:Function = ()=>true) =>
```

Comment:
This is going to cause type errors when you pass an arg to a function which doesn't take any. 


Also, it might be useful to define a type for the filter function and use it here. Also, it might be good to define a better type for the filter arg of this function:
```suggestion
type StorageClassFilter = (resource: StorageClass) => boolean;
const getUnorderedItems = (items, filter: StorageClassFilter = (resource) => true) =>
```